### PR TITLE
Salesforce states

### PR DESCRIPTION
**Pull request by lihanli**
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/8600

### CODE REVIEW

**Comment by jbalboni** [comment]
@lihanli Does SF have a list of countries we have to match, too?

**Comment by lihanli** [comment]
@jbalboni yea i added them just now

**Comment by jbalboni** [comment]
@lihanli It looks like they're not actually used in the schema definition, aside from the ones with states.

**Comment by lihanli** [comment]
@jbalboni now it's validating the country

File: src/schemas/VIC/schema.js [hunk 1]
```diff
@@ -1,13 +1,77 @@
...
+      postalCode: {
+        type: 'string',
+        maxLength: 10
+      }
+    }
+  }));
+  countryStateProperties.push({
+    properties: {
+      country: {
+        not: {
```
Comment (jbalboni) [inline src/schemas/VIC/schema.js]: We could probably get rid of this, right? Since the countriesWithNoState list would skip these?

**Comment by jbalboni** [comment]
@lihanli So can we add back the states that were removed from the US list? I think they're all in the SF settings, just marked as not visible.

**Comment by lihanli** [comment]
@jbalboni ok i added the hidden states from that xml file

**Pull request merged**